packages/scrollable-image/src/components/scroll-horizontal.js

```diff
@@ -106,8 +107,19 @@ class ScrollHorizontal extends React.PureComponent {
     // the wrapper’s height changes as well when the resize event is triggered.
     // The frequently changed height causes page jarring everytime when user changes scroll direction.
     // Therefore, only call `setLayout` function when the changed height is larger than url bar’s height.
-    const ignoredRange = 90
-    if (Math.abs(window.innerHeight - this.lastWindowHeight) > ignoredRange) {
+    const ignoredRange = 100
+    const canBeIgnored =
+      Math.abs(window.innerHeight - this.lastWindowHeight) < ignoredRange
+
+    // Currently, facebook in-app browser dispatches `resize` event when it opens up,
```

Comment:
Question 🙋‍♂️ :
I thought we have already handled resize event, why would facebook resize event cause the problem?

Reply:
Please checkout this [PR](https://github.com/twreporter/orangutan-monorepo/pull/75) and test in Facebook in-app browser.

This is my observation as an example:

![IMG_D6EB036EC68E-1](https://user-images.githubusercontent.com/6375655/96988299-bbf99980-1556-11eb-98a8-d57d59c6bb5d.jpeg)

When the in-app browser opens up, the `resized` event is always dispatched and this is a misbehavior.

The inappropriate event causes `this.contentWidth` [get incorrect value](https://github.com/twreporter/orangutan-monorepo/blob/master/packages/scrollable-image/src/components/scroll-horizontal.js#L114-L115).

On the other hand, there's no use cases that `resize` event need to be handled for Facebook in-app browser. So, I choose to exclude Facebook in-app browser in resize handler as the workaround.